include/h2o.h

```diff
@@ -644,27 +627,23 @@ struct st_h2o_context_t {
      * flag indicating if shutdown has been requested
      */
     int shutdown_requested;
-
+    /**
+     * link-list of h2o_conn_t
+     */
+    h2o_linklist_t _active_conns;
+    /**
+     * link-list of h2o_conn_t
+     */
+    h2o_linklist_t _idle_conns;
```

Comment:
Maybe use an unnamed struct like others?
```suggestion
    struct {
        h2o_linklist_t active, idle;
    }  _conns;
```